XenModel/InvisibleMessages.resx

```diff
@@ -138,6 +138,9 @@
   <data name="HOMEPAGE" xml:space="preserve">
     <value>http://www.citrix.com/xenserver</value>
   </data>
+  <data name="HOMEPAGE_FILENAME" xml:space="preserve">
+    <value>HomePage.mht</value>

```

Comment:
Also, please add HomePage.ja.mht and HomePage.zh-CN.mht files (just copies of the English ones at the moment), so the translators have something to translate.